package.xml

```diff
@@ -24,6 +24,7 @@
   <build_depend>tf2_geometry_msgs</build_depend>
 
   <run_depend>boost</run_depend>
+  <run_depend>libssl-dev</run_depend>
```

Comment:
I think what we need here is `build_export_depend` rather than `run_depend` [see](https://www.ros.org/reps/rep-0149.html#build-export-depend-multiple)

Reply:
this is still format 1. `run_depend` includes `exec_depend` and `build_export_depend` (which is not available for this format). I could just as well update the format here, though.